Decsys/Data/Entities/SurveyInstance.cs

```diff
@@ -16,24 +16,22 @@ public class SurveyInstance
         /// Create a SurveyInstance belong to a given Survey.
         /// </summary>
         /// <param name="surveyId">ID of the owning Survey.</param>
+        /// 
```

Comment:
can we lose these new lines? The XmlDoc (the `///` lines) should hug the class member:

```csharp
/// xmldoc comment
public SurveyInstance(...) { ... }
```